docs/contributing/blog-and-website-contributions.md

```diff
@@ -1,71 +1,71 @@
+- Faça o Commit e envie para o seu fork.
+- Abra um pull request da sua branch.
+  - Nós recomendamos usar o prefixo `docs`, ex. `docs/your-change` ou `docs-your-change`. ([Exemplo de PR](https://github.com/gatsbyjs/gatsby/commit/9c21394add7906974dcfd22ad5dc1351a99d7ceb#diff-bf544fce773d8a5381f64c37d48d9f12))
 
-- Clone [the Gatsby repo](https://github.com/gatsbyjs/gatsby/) and navigate to `/www`.
-- Run `yarn` to install all of the website's dependencies. ([Why Yarn?](/contributing/setting-up-your-local-dev-environment#using-yarn))
-- Run `npm run develop` to preview the blog at `http://localhost:8000/blog`.
-- The content for the blog lives in the `/docs/blog` folder. Make additions or modifications here.
-- Add your avatar image to `/docs/blog/avatars`.
-- Add your name to `/docs/blog/author.yaml`.
-- Add a new folder following the pattern `/docs/blog/yyyy-mm-dd-title`. Within this newly created folder, add an `index.md` file.
-- Add `title`, `date`, `author`, `excerpt`, and `tags` to the frontmatter of your `index.md`. You can [view existing tags](/blog/tags/), or [add a new one](https://github.com/gatsbyjs/gatsby/blob/master/www/src/data/tags-docs.js) if you feel your tag merits being its own tag, though we encourage you to use existing tags.
-- If you are cross posting your post, you can add `canonicalLink` for SEO benefits. You can check the other blog posts in `/docs/blog` for examples.
-- If your blog post contains images, add them to your blog post folder and reference them in your post's `index.md`.
-- Ensure any links to gatsbyjs.org are relative links - `/contributing/how-to-contribute/` instead of `https://gatsbyjs.org/contributing/how-to-contribute`.
-- Follow the [Style Guide](/contributing/gatsby-style-guide/#word-choice) to make sure you're using the appropriate wording.
-- Double check your grammar and capitalize correctly.
-- Commit and push to your fork.
-- Create a pull request from your branch.
-  - We recommend using a prefix of `docs`, e.g. `docs/your-change` or `docs-your-change`. ([PR example](https://github.com/gatsbyjs/gatsby/commit/9c21394add7906974dcfd22ad5dc1351a99d7ceb#diff-bf544fce773d8a5381f64c37d48d9f12))
+### Formato de postagem do Blog
 
-### Blog post format
-
-The following format can help you in creating your new blog content. At the top is "frontmatter": a fancy name for metadata in Markdown. The frontmatter for your post should include a title, date, singular author name (for now, we would welcome issues/PRs for this), and one or more tags. Your content will follow after the second set of dashes (`---`).
+O formato a seguir pode ajudá-lo a criar seu novo conteúdo para o blog. No topo está o "frontmatter"(cabeçalho): um nome _fancy_ para Metadados no Markdown. O cabeçalho para seu post deve incluir um título, data, um único autor (por hora, nós estamos aceitando issues/PR para isso) e uma ou mais tags. Seu conteúdo deve estar depois do segundo grupo de traços (`---`).
```

Comment:
Duas coisas, acho melhor adicionar um espaço entre "frontmatter" e "(cabeçalho)" e "um nome _fancy_ para Metadados no Markdown" to poderia traduzir para "um nome mais elegante para metadados no Markdown"